### ISSUE TITLE
Are there any QueryRendered Feature in flutterMap Package, Mapbox SDK?

### ISSUE THREAD

**Issue by maxchris100** [fleaflet/flutter_map#1573]
### What do you want implemented?

 QueryRendered Feature in flutterMap Package, Mapbox SDK?
 to highlight polygon layer

### Severity

Obtrusive: No workarounds are available, and this is essential to me

**Comment by ibrierley** [comment]
No there isn't. Devil is in the detail though, depending on the format and source of the polygons etc. It could be possible, i.e here's an example I did with a plugin ( click on a state to highlight ). https://fluttermap.dabbles.info/vector_outline/#/

There isn't anything within flutter_map itself though.

**Comment by maxchris100** [comment]
Yes this is what im looking for, how to get the highlight layer from the plugin? is it going to be in the next update flutter_map? 

> No there isn't. Devil is in the detail though, depending on the format and source of the polygons etc. It could be possible, i.e here's an example I did with a plugin ( click on a state to highlight ). https://fluttermap.dabbles.info/vector_outline/#/
> 
> There isn't anything within flutter_map itself though.

Yes this is what im looking for, how to get the highlight layer from the plugin? is it going to be in the next update flutter_map?

**Comment by ibrierley** [comment]
No, I'm pretty sure it won't be in the next flutter_map version. If you have the polys in geojson format you can have a play with the plugin at https://github.com/ibrierley/geojson_vector_slicer (it's a bit out of date with recent fluttermap versions, but pull requests welcome).
